sys/freebsd/dev_crypto.txt

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/time.h>
+include <fcntl.h>
+include <opencrypto/cryptodev.h>
+
+resource fd_crypto[fd]
+
+openat$crypto(fd const[AT_FDCWD], file ptr[in, string["/dev/crypto"]], flags flags[open_flags], mode const[0]) fd_crypto
+
+define CRIOGET	3221513060
+ioctl$CRIOGET(fd fd_crypto, cmd const[CRIOGET], arg ptr[inout, int32])
```

Comment:
The last parameter should be `arg ptr[out, fd_crypto]`.

Reply:
_IOWR is generally passed as inout as given in ioccom.h right? _IOW macro should take out then. Is that not how we check this?

Reply:
Ah, you are right. The in-value is ignored, but I guess it makes more sense to match the ioctl definition. Same for FINDDEV.